packages/api/src/hooks/useLandingCompany.ts

```diff
@@ -0,0 +1,33 @@
+import { useMemo } from 'react';
+import useFetch from '../useFetch';
+import useSettings from './useSettings';
+
+/** A custom hook that returns the available landing companies of the user's country. */
+const useLandingCompany = () => {
+    const { data: settings_data } = useSettings();
+    const { data, ...rest } = useFetch('landing_company', {
+        payload: { landing_company: settings_data.country_code || '' },
+        options: { enabled: Boolean(settings_data.country_code) },
+    });
+
+    const modified_data = useMemo(() => {
+        if (!data?.landing_company) return;
+        const { financial_company, gaming_company, virtual_company } = data.landing_company;
+        return {
+            /** Short code of financial landing company */
+            financial_company: financial_company?.shortcode,
+            /** Short code of gaming landing company */
+            gaming_company: gaming_company?.shortcode,
+            /** Short code of virtual landing company */
+            virtual_company,
+        };
+    }, [data]);
```

Comment:
Should we narrow down the `data` attribute dependency to `data?.landing_company` and `data?.settings_data`?

```suggestion
    }, [data?.landing_company, data?.settings_data]);
```

Reply:
I will make changes in this PR https://github.com/binary-com/deriv-app/pull/9785 and update this PR 👍